### DISCUSSION TITLE
GitHub Advice

### DISCUSSION THREAD

**Discussion by yvonneadam** [neuefische/questions#24] [Web] [answered]
Ich habe soeben in meinem Repo folgendes Fenster als Hinweis entdeckt: 
<<<<Your main branch isn't protected
Protect this branch from force pushing, deletion, or require status checks before merging. >>>>

Nur ein nettes Feature von GitHub oder hab ich was verkehrt gemacht? Und was sollte man folgend am besten alles anklickern? Danke

**Comment by stefankorder** [comment]
Wo erscheint dieser Hinweis denn genau? Kannst du vielleicht einen Screenshot hierzu machen?

**Comment by yvonneadam** [comment]
War direkt auf der Projekt-Startseite  (wo alle Dateien aufgelistet werden) zu sehen. Jetzt habe ich aber gemerged, sodass nur noch main vorhanden ist. Da war das Fenster verschwunden. Wenn es beim Öffnen einer neuen Branch/einem neuem PR wieder auftaucht, frage ich nochmal mit Foto ;)

**Reply by stefankorder** [reply]
Okay, super 👍🏻

**Comment by actyralf** [comment, marked as answer]
Dies Meldung könnt ihr ignorieren: In einem "echten" Setup wäre Eure `main` Branch geschützt, d.h. zum Beispiel, dass ihr selber da gar nichts rein mergen könnt - das würde dann z.B. Euer Teamlead machen. In der kostenfreien Github Version ist das jedoch meines Wissens nicht möglich.
